<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="ChoosePlatform">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Process">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ChooseOperation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="File">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>86</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Movie file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Directory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>86</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadImages">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowWindows">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show windows</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProcessingProgress">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="MorphologicalElementType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ImagesLoaded">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Images not Loaded</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lowLewel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Normalize">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="highLevel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetProcessed">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset processed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
